samples/SensorBLEDemo.js

```diff
@@ -90,11 +90,11 @@ ble.on('disconnect', function(clientAddress) {
 });
 
 var accel = new Accelerometer({
-    frequency: 100
+    frequency: 20
 });
 
 var gyro = new Gyroscope({
-    frequency: 100
+    frequency: 20
```

Comment:
Maybe define this as an updateFrequency variable at the top of the file? And maybe add a note there that it seems to work with 100 for production mode, but we need 20 for ashell?